src/app/static/src/app/components/header/DropDown.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
I factored out this component to be re-used by the file menu and the language menu.